samples/azure-quantum/entanglement-swap/README.md

```diff
@@ -0,0 +1,42 @@
+---
+page_type: sample
+author: adrianleh
+description: Entanglement Swapping using the Azure Quantum service
+ms.author: t-alehmann@microsoft.com
+ms.date: 08/02/2021
+languages:
+- qsharp
+- qiskit
+- python
+products:
+- qdk
+- azure-quantum
+---
+
+# Entanglement Swapping
+
+In this sample we will performing Entanglement Swapping.
+
+The idea is that Alice and Bob want to use quantum teleportation to share data.
+Though, they are too far apart to communicate directly.
+Hence, they will be use a number of middlemen to communicate.
+Each party will share an entangled pair with the parties next to them and teleport the information along the chain until it reaches Bob.
+Since this sample is fundamentally based on teleportation we use Quantinuum's mid-circuit measurement capability.
+
+This sample is a Q# jupyter notebook targeted at IonQ and Quantinuum machines.
```

Comment:
```suggestion
This sample is a Q# Jupyter notebook targeted at IonQ and Quantinuum machines.
```
Does this run on IonQ? I thought it used intermediate measurements? It may also be good to mention that the sample is available as a Qiskit notebook.